packages/browser-agent-core/src/common/util/obfuscate.js

```diff
@@ -55,17 +52,17 @@ export function validateRules(rules) {
   var invalidRegexDetected = false
   for (var i = 0; i < rules.length; i++) {
     if (!('regex' in rules[i])) {
-      if (console && console.warn) console.warn('An obfuscation replacement rule was detected missing a "regex" value.')
+      if (console && console.warn) console.warn('New Relic: An obfuscation replacement rule was detected missing a "regex" value.')
       invalidRegexDetected = true
     } else if (typeof rules[i].regex !== 'string' && !(rules[i].regex.constructor === RegExp)) {
-      if (console && console.warn) console.warn('An obfuscation replacement rule contains a "regex" value with an invalid type (must be a string or RegExp)')
+      if (console && console.warn) console.warn('New Relic: An obfuscation replacement rule contains a "regex" value with an invalid type (must be a string or RegExp)')
       invalidRegexDetected = true
     }
 
     var replacement = rules[i].replacement
     if (replacement) {
       if (typeof replacement !== 'string') {
-        if (console && console.warn) console.warn('An obfuscation replacement rule contains a "replacement" value with an invalid type (must be a string)')
+        if (console && console.warn) console.warn('New Relic: An obfuscation replacement rule contains a "replacement" value with an invalid type (must be a string)')
```

Comment:
Just a question on this. What happens if the obfuscation rule is invalid? Do we still capture the data and send it to NR1? We might want to consider throwing an error here and crashing the agent to prevent PII/PHI from possibly making it to NR1.

Reply:
I think we took the approach of "its on them to configure it correctly" and we will give them messaging to aide in that but we didn't want to kill all data collection.  worth revisiting